eclipse/plugin/src/nl/tudelft/watchdog/eclipse/logic/ui/listeners/EclipseMarkupModelListener.java

```diff
@@ -0,0 +1,213 @@
+package nl.tudelft.watchdog.eclipse.logic.ui.listeners;
```

Comment:
Can we break this apart into its class constiuents and move them to their own package?

Reply:
They are part of the whole implementation and not intended to be used outside of this class. Moreover, they reference methods/fields in the outer class, so this would make the logic more complicated.